src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -679,21 +683,21 @@ using namespace ngraph::runtime;
 
         // create output alias map
         /*
-        size_t output_index = 0;
-        unordered_map<descriptor::TensorView*, vector<size_t>> output_alias_map;
-        vector<size_t> aliases;
-        for (size_t i = 0; i < current_function->get_output_size(); ++i)
-        {
-            shared_ptr<Node> op = current_function->get_output_op(i);
-            shared_ptr<descriptor::TensorView> otv = op->get_output_tensor_view();
-            vector<size_t>& al = output_alias_map[otv.get()];
-            al.push_back(output_index);
-            if (al.size() > 1)
-            {
-                aliases.push_back(output_index);
-            }
-            output_index++;
-        }
+          size_t output_index = 0;
```

Comment:
needs style check.

Reply:
@jbobba it's all commented out. I'd prefer a #if 0 ... #endif with a comment since the indentation doest get messed up like this and you don't need to worry about nested comments.

Reply:
Thanks for catching that. Should I just remove it? Anyone want to make a case for why it should be there?

Reply:
Remove it.